save-ui-version.sh

```diff
@@ -1,16 +1,16 @@
 #!/usr/bin/env bash
 
-GIT_DIR=$(echo $(git rev-parse --git-dir)/../src)
-VERSION_JSON="$GIT_DIR/views/About/version.json"
+# Save the git commit SHA and date from the current branch as JSON in a file
+# imported by the About page.
+
+VERSION_JSON="$(git rev-parse --show-toplevel)/src/views/About/version.json"
 
 commit_sha=$(git rev-parse --short HEAD)
 commit_date=$(git log -1 --format="%at" | xargs -I{} date -u -d @{} +"%Y/%m/%d %H:%M:%S %Z")
 
 cat << EOF > $VERSION_JSON
-[
-  {
-    "commit": "$commit_sha",
-    "date": "$commit_date"
-  }
-]
+{
+  "commit": "$commit_sha",
+  "date": "$commit_date"
+}
```

Comment:
I don't think I'm the correct person to ensure that the git commands are correct. Perhaps @russtuck or @EjPlatzer can confirm.

Reply:
The script should work fine (it does on development computers) but won't know for sure the path the version.json file is correct until script is run during deployment.